src/Web/Sfa.Das.Sas.Shared.Components/Orchestrators/TrainingProviderOrchestrator.cs

```diff
@@ -92,28 +96,36 @@ public async Task<TrainingProviderSearchFilterViewModel> GetSearchFilter(Trainin
         public async Task<TrainingProviderDetailsViewModel> GetDetails(TrainingProviderDetailQueryViewModel detailsQueryModel)
         {
 
-            var response = await _mediator.Send(new ApprenticeshipProviderDetailQuery() { UkPrn = Convert.ToInt32(detailsQueryModel.Ukprn), ApprenticeshipId = detailsQueryModel.ApprenticeshipId, ApprenticeshipType = detailsQueryModel.ApprenticeshipType, LocationId = detailsQueryModel.LocationId });
+            var cacheKey = detailsQueryModel.Ukprn + detailsQueryModel.LocationId + detailsQueryModel.ApprenticeshipId;
```

Comment:
this key needs to be a bit safer, there is a chance multiple objects could have the same key as there is no seperation between the values, i suggest something like 
`var cacheKey = $"providerDetails-{detailsQueryModel.Ukprn}-{detailsQueryModel.LocationId}-{ApprenticeshipId}";`